### PR TITLE
Mountain component for 7summit backend

### PR DESCRIPTION
**Pull request by ivan-berezhnov**

### CODE REVIEW

File: docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue [hunk 1]
```diff
@@ -24,82 +24,193 @@
...
+        }
+
+        this.currentExercise = exercise;
+        this.exerciseModalVisible = true;
+      },
+      onExerciseClosed: function () {
+        this.exerciseModalVisible = false;
+
+        if (this.fullyCompletedTodayExercises()) {
+          BadgeHelper.create_hidden_image(this.summit.game_nid);
```
Comment (anpolimus) [inline docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue:172]: This code is not needed here. It creates a badge for the hidden picture.

File: docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue [hunk 2]
```diff
@@ -24,82 +24,193 @@
...
+          window.location = window.location.origin + '/user';
```
Comment (anpolimus) [inline docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue:173]: Once the user has finished all tasks for this mountain, he should be redirected to the all mountains route, not a user screen.

File: docroot/modules/custom/twelve_app/src/Plugin/Block/SevenSummits.php [hunk 3]
```diff
@@ -73,6 +73,9 @@ function getSummits() {
       ->get($this->getGameConfigurationName())
       ->get('items');
 
+    $imageStyleStorage = \Drupal::service('entity_type.manager')->getStorage('image_style');
```
Comment (anpolimus) [inline docroot/modules/custom/twelve_app/src/Plugin/Block/SevenSummits.php:76]: ImageStyle::load()

Comment (ivan-berezhnov) [reply docroot/modules/custom/twelve_app/src/Plugin/Block/SevenSummits.php]: I have fixed

File: docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue (hunk 2 above)
Comment (ivan-berezhnov) [reply docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue]: I have fixed

File: docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue (hunk 1 above)
Comment (ivan-berezhnov) [reply docroot/modules/custom/twelve_app/js/seven_summits/views/Mountain.vue]: I have fixed

**Pull request merged**